### PR TITLE
Documents Simplified Chinese Localization / 文档的中文本地化

### PR DESCRIPTION
**Pull request by MaikoTan**
This PR aims to localize all the documents to Chinese.

You can see the preview version [here](https://github.com/MaikoTan/cactbot/blob/docs-zh-cn/docs/zh-CN/)

### CODE REVIEW

File: docs/zh-CN/CactbotCustomization.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
+# Cactbot自定义教程
+
+- [使用cactbot配置界面](#使用cactbot配置界面)
+- [用户文件夹概览](#用户文件夹概览)
+- [设置您自己的用户文件夹](#设置您自己的用户文件夹)
+- [样式自定义](#样式自定义)
+- [Raidboss触发器自定义](#Raidboss触发器自定义)
+  - [例1：改变输出文本](#例1：改变输出文本)
```
Comment (panicstevenson) [inline docs/zh-CN/CactbotCustomization.md:8]:
```suggestion
  - [例1：改变输出文本](#例1改变输出文本)
```

Comment (MaikoTan) [reply docs/zh-CN/CactbotCustomization.md:8]: Oh, you're right! Thank you!

**Comment by quisquous** [comment]
Is this still a draft, or is this ready to merge?

**Comment by MaikoTan** [comment]
For this single file it is completed. Only lacking the links to its English version and vise versa.

But I and Echoring are working on other documents too, should we seperate each file on its own PR, or let this single PR for all of the documents localization ?

**Comment by quisquous** [comment]
Thanks! I think having separate PRs would be good.

**Pull request merged**